docs/visualization/visualize_2p_responses.ipynb

```diff
@@ -0,0 +1,1053 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    time_start = 0
</pre><p>Could you choose a unit that has clearer response to some stimulus epochs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/128/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>